soh/soh/Enhancements/randomizer/randomizer_item_tracker.cpp

```diff
@@ -986,6 +1018,8 @@ void DrawItemTracker(bool& open) {
             SohImGui::EnhancementCheckbox("Display \"Ammo/MaxAmo\"", "gItemTrackerAmmoDisplay");
             SohImGui::EnhancementCheckbox("Randomizer colors for Songs", "gItemTrackeSongColor");
             SohImGui::Tooltip("Will dispaly non-warp songs with randomizer\ncolors instead of pure white");
+            SohImGui::EnhancementCheckbox("Personnal notes space", "gItemTrackerNotes");
+            SohImGui::Tooltip("Adds a personal space under the item list to\n write eyour own notes.");
```

Comment:
Personnal -> Personal
eyour -> your

Also, suggested tooltip change: "Adds a textbox under the item tracker to keep your own notes in."

Reply:
Indeed, it was late for me when I pushed that and didn't paid attention to the typing, done ^-^

Reply:
Welp I guess I still suck at Github. Just wanted to approve the text changes, lol. Oh well.